src/sentry/integrations/slack/message_builder/base/__init__.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC
+
+
+class AbstractMessageBuilder(ABC):
+    """TODO(mgaeta): Make this a common interface with EMAIL and MSTEAMS!"""
```

Comment:
does this need to be done now or is it for later?

Reply:
This can be done much later. I'm leaving the note here to remind myself.